src/main/java/com/softserveinc/ita/pageobjects/BasketModal.java

```diff
@@ -1,5 +1,12 @@
 package com.softserveinc.ita.pageobjects;
 
+import static com.codeborne.selenide.Selenide.$x;
+
 public class BasketModal extends BasePage<BasketModal> {
 
+    public String getProductTitle() {
+        String productTitleSelector = "//a[@class = 'cart-product__title']";
```

Comment:
you don't need a separate variable for the selector, insert it into the selenide element